pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/ElasticSearchUtil.java

```diff
@@ -706,17 +711,22 @@ private static void updateOrCreateMap(@NonNull Object id, @NonNull String object
         }
     }
 
-    public static void updateRequest(@NonNull String ddpParticipantId, String index, Map<String, Object> objectsMapES) throws IOException {
+    public static void updateRequest(@NonNull String ddpParticipantId, String index, Map<String, Object> objectsMapES) {
         initialize();
-        updateRequest(ddpParticipantId, index, objectsMapES, client);
+        doUpdate(ddpParticipantId, index, objectsMapES, client);
     }
 
-    private static void updateRequest(@NonNull String ddpParticipantId, String index, Map<String, Object> objectsMapES,
-                                      RestHighLevelClient client) throws IOException {
+    private static void doUpdate(@NonNull String ddpParticipantId, String index, Map<String, Object> objectsMapES,
+                                 RestHighLevelClient client) {
         String participantId = ParticipantUtil.isGuid(ddpParticipantId) ? ddpParticipantId :
                 getParticipantESDataByAltpid(client, index, ddpParticipantId).getProfile().map(Profile::getGuid).orElse(ddpParticipantId);
         UpdateRequest updateRequest =
-                new UpdateRequest().index(index).type("_doc").id(participantId).doc(objectsMapES).docAsUpsert(true).retryOnConflict(5);
+                new UpdateRequest().index(index).type("_doc")
+                        .id(participantId)
+                        .doc(objectsMapES)
+                        .docAsUpsert(true)
+                        .setRefreshPolicy(WriteRequest.RefreshPolicy.IMMEDIATE)
```

Comment:
Could you say more about the choice of refresh policy here?  In the PR description, I think something got cut off:

> Updated the base ES update method ElasticSearchUtil.doUpdate to use IMMEDIATE refresh policy, since we've seen